<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>609</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>789</width>
    <height>347</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>7899</width>
    <height>3477</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QtCompressor - 測試檔案壓縮極限</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/rank.png</normaloff>:/images/rank.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.970000000000000</double>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>開始測試 (快捷鍵：Ctrl+S)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>開始測試 (快捷鍵：Ctrl+S)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/fire.png</normaloff>:/images/fire.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>可直接拖曳測試檔案或資料夾至此</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>選擇輸出暫存檔位置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>輸出位置</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/save-file.png</normaloff>:/images/save-file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>541</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>列出詳細執行過程</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>選擇檔案</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open-file.png</normaloff>:/images/open-file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>145</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>模式選擇</string>
   </property>
   <widget class="QRadioButton" name="mode3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>使用PAQ8PXD測試</string>
    </property>
    <property name="text">
     <string>極限模式 (慎用)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mode1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>使用WinRAR、7-Zip、UPX測試</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string>標準模式</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mode2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>使用WinRAR、7-Zip、UHARC、UPX測試</string>
    </property>
    <property name="text">
     <string>進階模式</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>103</y>
     <width>791</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="delTestFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>不保留測試檔案</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="keepTestFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>78</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>輸出位置須保留至少2倍測試檔案大小以上</string>
    </property>
    <property name="text">
     <string>保留最小測試檔案</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="keepAllTestFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>118</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>輸出位置須保留至少13倍測試檔案大小以上</string>
    </property>
    <property name="text">
     <string>保留所有測試檔案</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
